server.go

```diff
@@ -595,7 +597,7 @@ func (s *Server) unregister() error {
 	return s.multicastResponse(resp, 0)
 }
 
-func (s *Server) appendAddrs(list []dns.RR, ifIndex int) []dns.RR {
+func (s *Server) appendAddrs(list []dns.RR, ttl uint32, ifIndex int) []dns.RR {
```

Comment:
Might sound very sensitive, but I'd like to see a `uint8 ` or just `uint`, as specified in the IP RFCs.

Reply:
Just saw that in the structs it is also wrongly defined as `uint32. So ignore this comment. Needs a change all over.